lib/server/controllers/lhcPeriodStatistics.controller.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+const Joi = require('joi');
+const { ApiConfig } = require('../../config/index.js');
+const { DtoFactory } = require('../../domain/dtos/DtoFactory');
+const { lhcPeriodStatisticsService } = require('../services/lhcPeriod/LhcPeriodStatisticsService.js');
+const { dtoValidator } = require('../utilities/dtoValidator.js');
+const { countedItemsToHttpView } = require('../utilities/countedItemsToHttpView.js');
+const { updateExpressResponseFromNativeError } = require('../express/updateExpressResponseFromNativeError');
+const PaginationDto = require('../../domain/dtos/PaginationDto.js');
+
+/**
+ * List All LHC Periods with statistics
+ * @param {Object} req express HTTP request object
+ * @param {Object} res express HTTP response object
+ * @returns {undefined}
```

Comment:
We return `void` not `undefined`
In controllers you can omit params and return and put an eslint ignore line (simply put the line on what the endpoint does

Reply:
done